app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package com.example.nvdovin.weatherapp.domain.service;
+
+import android.content.Context;
+
+import com.example.nvdovin.weatherapp.data.SortQueryBuilder;
+import com.example.nvdovin.weatherapp.data.dao.CityDao;
+import com.example.nvdovin.weatherapp.data.dao.DaoSession;
+import com.example.nvdovin.weatherapp.data.model.City;
+
+import org.greenrobot.greendao.Property;
+import org.greenrobot.greendao.query.QueryBuilder;
+import org.greenrobot.greendao.query.WhereCondition;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CityServiceTest {
+
+    @Mock
+    DaoSession daoSession;
+    @Mock
+    QueryBuilder queryBuilder;
+    @Mock
+    City city;
+    @Mock
+    CityDao cityDao;
+    @Mock
+    WhereCondition whereCondition;
+    @Mock
+    SortQueryBuilder sortQueryBuilder;
+    @Mock
+    Property property;
+    @Mock
+    Context context;
+    @Mock
+    List list;
+    private CityService cityService;
+
+    @Before
+    public void setUp() throws Exception {
+        cityService = new CityService(daoSession, context);
+    }
+
+    @Test
+    public void testInsert() throws Exception {
+
+        when(daoSession.getCityDao()).thenReturn(cityDao);
+
+        cityService.insert(city);
+
+        verify(daoSession).getCityDao();
+    }
+
+    @Test
+    public void testGetCityById() throws Exception {
+
+        initBaseCondition();
+
+        when(queryBuilder.where(
+                any(WhereCondition.class),
+                any(WhereCondition.class)
+        )).thenReturn(queryBuilder);
+        when(queryBuilder.unique()).thenReturn(city);
+
+        cityService.getCityById(1L);
+
+        verifyBaseCondition();
+
+        verify(queryBuilder).where(any(WhereCondition.class));
+        verify(queryBuilder).unique();
+    }
+
+    @Test
+    public void testLoadSortedCitiesOrderAsc() throws Exception {
+
+        initBaseCondition();
+
+        doReturn(false).when(sortQueryBuilder).isAscending();
```

Comment:
why don't keep it consistent, and call here also when from Mokito? 
I mean to have the same as below `when(sortQueryBuilder.isAscending()).thenReturn(false);`